keras_nlp/utils/preset_utils_test.py

```diff
@@ -0,0 +1,215 @@
+# Copyright 2023 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import os
+
+from keras_nlp.backend import keras
+from keras_nlp.models import AlbertMaskedLM
+from keras_nlp.models import BartSeq2SeqLM
+from keras_nlp.models import BertMaskedLM
+from keras_nlp.models.albert import albert_presets
+from keras_nlp.models.bart import bart_presets
+from keras_nlp.models.bert import bert_presets
+from keras_nlp.tests.test_case import TestCase
+from keras_nlp.utils import preset_utils
+
+
+class PresetUtilsTest(TestCase):
+    def test_albert_sentencepiece_preset_saving(self):
```

Comment:
I would actually recommend against this because of differences between the different tokenizer types. BytePair uses a vocabulary.json, SentencePair has proto logic, and WordPair just has a plain vocabulary.txt.

Reply:
Nvm, managed to parameterize successfully!